src/colors.rs

```diff
@@ -78,6 +79,7 @@ macro_rules! colors {
 
         $(
             /// A color for use with [`OwoColorize`](crate::OwoColorize)'s `fg` and `bg` methods.
+            #[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
```

Comment:
What is the representation of this? I would want this to be serialized to a string representation of this color (in kebab case, probably), and then deserialization checks for that. Is this what serde does?

Reply:
Unit structs are deserialized using the string representation of the struct. So yes, this can be `kebab-case`